docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,171 @@ description: Next.js supports built-in image optimization, as well as third part
+>
+>**Normalize your images**
+>
+>If you're serving images from a source that you control, consider modifying your image pipeline to normalize the images to a specific size.
+>
+>**Modify your API calls**
+>
+>If your application is retrieving image URLs using an API call (such as to a CMS), you may be able to modify the API call to return the image dimensions along with the URL.
 
-The expiration (or rather Max Age) is defined by the upstream server's `Cache-Control` header.
+If none of the suggested methods works for sizing your images, the `next/image` component is designed to work well on a page alongside standard `<img>` elements. 
 
-If `s-maxage` is found in `Cache-Control`, it is used. If no `s-maxage` is found, then `max-age` is used. If no `max-age` is found, then [`minimumCacheTTL`](#minimum-cache-ttl) is used.
+## Styling
 
-You can configure [`minimumCacheTTL`](#minimum-cache-ttl) to increase the cache duration when the upstream image does not include `max-age`.
+Styling the Image component is not that different from styling a normal `<img>` element, but there are a few guidelines to keep in mind:
 
-You can also configure [`deviceSizes`](#device-sizes) and [`imageSizes`](#device-sizes) to reduce the total number of possible generated images.
+**Pick the correct layout mode**
 
-## Advanced
+The image component has several different [layout modes](/docs/api-reference/next/image.md#layout) that define how it is sized on the page. If the styling of your image isn't turning out the way you want, consider experimenting with other layout modes.
 
-The following configuration is for advanced use cases and is usually not necessary. If you choose to configure the properties below, you will override any changes to the Next.js defaults in future updates.
+**Target the image with className, not based on DOM structure**
 
-### Device Sizes
+Regardless of the layout mode used, the Image component will have a consistent DOM structure of one `<img>` tag wrapped by exactly one `<div>`. For some modes, it may also have a sibling `<div>` for spacing. These additional `<div>` elements are critical to allow the component to prevent layout shifts.
 
-In some cases, where you know the expected device widths from the users of your website, you can specify a list of device width breakpoints using the `deviceSizes` property. These widths are used when the [`next/image`](/docs/api-reference/next/image.md) component uses `layout="responsive"` or `layout="fill"` so that the correct image is served for the device visiting your website.
+The easiest way to style the inner `<img>` rendered by the image element is to **apply a class to your images using the `className` property** on the Image element and then target that class with your styles. The value of className will be automatically applied to the underlying `<img>` element.
```

Comment:
```suggestion
The only way to style the inner `<img>` is to **apply a class to your images using the `className` property** on the Image component and then target that class with your styles. The value of `className` will be automatically applied to the underlying `<img>` element.
```

Reply:
Is this to correct to say that classname is the _only_ way to style the img? Or should we say it's the "best" way or something, because you definitely _can_ style the <img> in other ways, it's just not a good idea.

Reply:
I suppose the only other way is to target `img` in global CSS.

How about this:

```suggestion
The best way to style the inner `<img>` is to set the `className` prop on the Image component to the value of an imported [CSS Module](/docs/basic-features/built-in-css-support.md#adding-component-level-css). The value of `className` will be automatically applied to the underlying `<img>` element.

Alternatively, you can import a [global stylesheet](/docs/basic-features/built-in-css-support#adding-a-global-stylesheet) and manually set the `className` prop to the same name used in the global stylesheet.

You cannot use [styled-jsx](/basic-features/built-in-css-support#css-in-js) because its scoped to the current component.

You cannot use the `style` prop because the `<Image>` component does not pass it through to the underlying `<img>`.
```

Reply:
Could we swap "best" for "recommended" ?